### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anoblet**
Fix https://github.com/open-wc/open-wc/issues/866

### CODE REVIEW

File: packages/es-dev-server/README.md [hunk 1]
```diff
@@ -137,9 +137,12 @@ index.html
 
 If you run `es-dev-server` regularly from the root of this project, you can access your app at `/` or `/index.html` in the browser.
 
+</details>
+
 ### index.html in a subfoolder
 
-If you move your `index.html` inside a subfolder:
```
Comment (anoblet) [inline packages/es-dev-server/README.md]: This should go back in to the folded content

File: packages/es-dev-server/README.md [hunk 2]
```diff
@@ -122,11 +122,11 @@ In addition to the command-line flags, the configuration file accepts these addi
 
 Click read more to view different strategies for setting up your project's folder structure.
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
Outside of that one requirement, however, `es-dev-server` does not have any opinions on how you should scaffold your project. The following are examples of a variety of different suggested strategies for setting up your project's folder structure.
```

File: packages/es-dev-server/README.md [hunk 3]
```diff
@@ -122,11 +122,11 @@ In addition to the command-line flags, the configuration file accepts these addi
...
 
+### index.html in root
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
### index.html in the Root
```

File: packages/es-dev-server/README.md [hunk 4]
```diff
@@ -137,10 +137,15 @@ index.html
 
 If you run `es-dev-server` regularly from the root of this project, you can access your app at `/` or `/index.html` in the browser.
 
+</details>
+
 ### index.html in a subfoolder
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
### index.html in a Subfolder
```

File: packages/es-dev-server/README.md [hunk 5]
```diff
@@ -195,9 +205,14 @@ es-dev-server --app-index packages/my-package/index.html --open
 
 This is the same approach as serving an index.html in a subdirectory, so the section above applies here as well.
 
+</details>
+
 ### Base element
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
### Base Element
```

File: packages/es-dev-server/README.md [hunk 6]
```diff
@@ -122,11 +122,11 @@ In addition to the command-line flags, the configuration file accepts these addi
 
 Click read more to view different strategies for setting up your project's folder structure.
 
+### index.html in root
+
 <details>
   <summary>Read more</summary>
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
  <summary>The simplest setup, making sure that all files are accessible, is to place your index.html at the root of your project</summary>
```

File: packages/es-dev-server/README.md [hunk 7]
```diff
@@ -122,11 +122,11 @@ In addition to the command-line flags, the configuration file accepts these addi
...
 
-### index.html in root
-
 The simplest setup where all files are accessible is to place your index.html at the root of your project:
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
Consider this example directory structure in the web root:
```

File: packages/es-dev-server/README.md [hunk 8]
```diff
@@ -137,10 +137,15 @@ index.html
 
 If you run `es-dev-server` regularly from the root of this project, you can access your app at `/` or `/index.html` in the browser.
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
If you run the `es-dev-server` command from the root of the project, you can access your app at `/` or `/index.html` in the browser.
```

File: packages/es-dev-server/README.md [hunk 9]
```diff
@@ -137,10 +137,15 @@ index.html
...
 
+</details>
+
 ### index.html in a subfoolder
 
 If you move your `index.html` inside a subfolder:
 
+<details>
+  <summary>Read more</summary>
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
  <summary>Use the `--open` parameter for when you'd like to keep you index.html in a subfolder.</summary>
```

File: packages/es-dev-server/README.md [hunk 10]
```diff
@@ -166,8 +171,13 @@ Now your `index.html` is accessible at `/` or `/index.html`. However, the dev se
 
 If you want your index in a subfolder without this being visible in the browser URL, you can set up a file rewrite rule. [Read more here](#rewriting-file-requests)
 
+</details>
+
 ### Monorepos
 
+<details>
+  <summary>Read more</summary>
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
  <summary>Use `--app-index` or `--root-dir` when your index.html and web root are in different places, e.g.. in a monorepo setup.</summary>
```

File: packages/es-dev-server/README.md [hunk 11]
```diff
@@ -195,9 +205,14 @@ es-dev-server --app-index packages/my-package/index.html --open
 
 This is the same approach as serving an index.html in a subdirectory, so the section above applies here as well.
 
+</details>
+
 ### Base element
 
-You can set up a `<base href="">` element to modify how files are resolved relatively to your index.html. You can be very useful when your index.html is not at the root of your project.
+<details>
+  <summary>Read more</summary>
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
  <summary>Use platform features to specify your web root, e.g. in <abbr title="Single Page Applications">SPAs</abbr></summary>
```

File: packages/es-dev-server/README.md [hunk 12]
```diff
@@ -195,9 +205,14 @@ es-dev-server --app-index packages/my-package/index.html --open
...
+
+You can set up a `<base href="">` element to modify how files are resolved relatively to your index.html. This can be very useful when your index.html is not at the root of your project.
 
 If you use SPA routing, using a base element is highly recommended. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)
```
Comment (bennypowers) [inline packages/es-dev-server/README.md]:
```suggestion
If you use <abbr title="Single Page Application">SPA</abbr> routing, using a base element is highly recommended. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)
```

**Pull request merged**